packages/liveblocks-core/src/room.ts

```diff
@@ -2962,48 +2962,36 @@ export function createRoom<
 
     const PAGE_SIZE = 50;
 
-    const response = await fetchCommentsApi(
+    const result = await fetchCommentsJson<{
```

Comment:
Noticed we have a helper method `fetchCommentsJson` that checks the status in case of !response.ok and throws `CommentsApiError` with response status and message, so instead used it so I could check for response status from the store.

https://github.com/liveblocks/liveblocks/blob/7e6c8cb485f9a827523c5d0402bc009b8430ea5b/packages/liveblocks-core/src/room.ts#L2866-L2902